### PR TITLE
Depend on WordPressKit ~> 6.0

### PR DESCRIPTION
**Pull request by crazytonyli**
Bump the WordPressKit dependency from v5 to v6, so that it can be upgraded on WordPress-iOS (which I have [a draft PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/19908) for).

---

- [x] I have considered if this change warrants release notes and have added them to the appropriate section in the `CHANGELOG.md` if necessary.

### CODE REVIEW

**Review by mokagio — CHANGES_REQUESTED** [review]
Unfortunately, because I was AFK on Friday and didn't get a chance to look at this PR, [version 5.1.0 was released](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/730) without this being merged. 

I'm guessing we'll need to ship a new version now. 5.2.0.

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,6 +48,12 @@ _None._
 
 _None._
 
+## [5.2.0](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/releases/tag/5.0.0)
+
+### Internal Changes
+
+- Change minimum version of WordPressKit to 6.0.
+
 ## [5.1.0]
```
Comment (mokagio) [inline CHANGELOG.md]: Taking the chance to also fix this broken link:

```suggestion
## [5.1.0](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/releases/tag/5.1.0)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -48,6 +48,12 @@ _None._
 
 _None._
 
+## [5.2.0](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/releases/tag/5.0.0)
```
Comment (mokagio) [inline CHANGELOG.md]:
```suggestion
## [5.2.0](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/releases/tag/5.2.0)
```

**Review by mokagio — APPROVED** [review]
Good to go. I've taken the liberty of applying my link fixes myself to avoid them being missed before merging.

File: CHANGELOG.md (hunk 2 above)
Comment (crazytonyli) [reply CHANGELOG.md]: Ops, copy-and-paste error, thanks for fixing it 😄

**Pull request merged**